src/components/AddLeanerModal.tsx

```
@@ -205,12 +210,26 @@ const AddLearnerModal: React.FC<AddLearnerModalProps> = ({
+            dob:apiBody?.dob,
+            gender:apiBody?.gender
           };
           const customFields = apiBody.customFields;
           const object = {
             userData: userData,
             customFields: customFields,
           };
+
+          if(learnerEmailId===userData.email)
+          {
+            delete userData.email;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)